tools/benchmark.py

```diff
@@ -87,7 +88,7 @@ def timed(name, function, args=[], setup=None):
         function(*args)
         times.append(time.time() - start)
     best_time = min(times)
-    print "%s%d" % (name + (60 - len(name)) * ".", per_trial / best_time)
+    print("{:s}{:d}".format(name + (60 - len(name)) * ".", per_trial / best_time))
```

Comment:
This should be `{0}{1}` to support python 2.6. or you can just use `%` formatting as before which is valid on all python versions.